i18n/bg.toml

```diff
@@ -0,0 +1,201 @@
+# Translations for English
```

Comment:
Something tells me that there must be something else))

Reply:
oops fixed with latest commit